source/industry/UR.rst

```diff
@@ -0,0 +1,166 @@
+  
+
+Trajectory composition
+----------------------
+Programming is done with move instructions (movement types) that move the robot to specified targets.
+
+* Linear Cartesian motions
+   **MoveL**
+   
+   * tool moves in a straight line. To keep moving linearly between waypoints each joint performs a more complicated motion. 
+   * parameters (pose, a=1.2, v=0.25, t=0, r=0):
+   
+      * pose: target pose
+      * a: tool acceleration in m/s\ :sup:`2`
+      * v: tool speed in m/s
+      * t: time in s
+      * r: blend radius in m
+
+* Circular motions
+   **MoveC**
+   
+   * tool moves on a circular arc segment to from current pose to target pose. Path point `pose_via` defines the arch's shape
+   * parameters (pose_via, pose_to, a=1.2, v=0.25, r=0, mode=0):
+   
+      * pose_via: path point
+      * pose_to: target pose
+      * a: tool acceleration in m/s\ :sup:`2`
+      * v: tool speed in m/s
+      * r: blend radius (of target pose) in m
+      * mode: (0: Unconstrained / 1: Fixed mode)
+
+      
+* Joint space interpolation
+   **MoveJ**
+   
+   * tool moves in a curved path interpolated in joint space. Each joint reaches location simultaneously. Preferred motion if a high TCP speed is desired. 
+   * parameters (q, a=1.4, v=1.05, t=0, r=0):
+   
+      * q: joint positions
+      * a: joint acceleration of leading axis in rad/s\ :sup:`2`
+      * v: joint speed of leading axisin rad/s
+      * t: time in s
+      * r: blend radius in m
+
+* Additional
+   **MoveP** 
+   
+   * tool moves linearly with constant speed with circular blends. Command can be extendend by a Circle move consisting of two waypoints. 
+   * parameters (pose, a=1.2, v=0.25, r=0):
+   
+      * pose: target pose
+      * a: tool acceleration in m/s\ :sup:`2`
+      * v: tool speed in m/s
+      * r: blend radius in m
+
+
+
+
+Waypoint representation
+-----------------------
+Individually taught points have the following representation:
+
+.. code-block:: yaml
+
+  X
+  Y
+  Z
+  Rx (roll)
+  Ry (pitch)
+  Rz (yaw)
+  
+
+Trajectory parameterization and execution
+-----------------------------------------
+
+Specification of velocity
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The robot's speed is defined in form of an argument by the move command.
+   
+    * For *MoveJ* in deg/s:  **maximum joint speed**
+    * For *MoveL* in mm/s:  **desired tool speed**
+      
+Global specification of velocity are done sparately for joint and TCP speed. The limits, which depend on the robot version, are stated in the table below. The actual speed limits also depend on the robot configuration.
+      
+     ===========  ===========================     ===========
+     Function     Description                     Limit
+     ===========  ===========================     ===========
+     Joint speed  Max. angular joint speed        30 ◦/s
+     TCP speed    Max. speed of the robot TCP     5000 mm/s
+     ===========  ===========================     ===========
+
```

Comment:
```suggestion
.. [#] An exception is the wrist3 of UR3s with max. angular speed of 360°/s.
```

Reply:
I think, I would prefer, if you could either edit your original suggestion or make a commit. This little suggestion bits make it hard to follow up.

Reply:
Fixed in bfc6a8f